apps/src/blockly/addons/contextMenu.js

```diff
@@ -1,6 +1,14 @@
 import GoogleBlockly from 'blockly/core';
 import msg from '@cdo/locale';
 
+const blocklyTheme = 'blocklyTheme';
```

Comment:
nit: can you make the constant names in all caps so it's clear they are constants?